microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -564,7 +564,11 @@ def _parallel_cleanup(node: Node) -> None:
         if not interface.is_enabled_sriov():
             interface.switch_sriov(enable=True, wait=False, reset_connections=True)
             # cleanup temporary hugepage and driver changes
-        node.reboot()
+        try:
+            node.reboot(time_out=60)
+        except LisaException:
+            node.log.debug("Timeout during cleanup reboot. Marking node for deletion.")
+            node.mark_dirty()
```

Comment:
Should the test case fail, if reboot failed? The exception is caught, so the test case won't be failed.

Reply:
I care about the DPDK test results. If the test passed and there's a timeout in the cleanup path I think it's better to just mark the node dirty and move on. If there was a crash won't LISA still post the serial console log to the result?

Reply:
This change is the result of multiple instances of me going to check a failed test result and seeing the test was fine, LISA just wasn't able to reconnect after rebooting during cleanup for whatetever reason.